package com.test.blog.service.exceptions;

public class UserNotFoundException extends RuntimeException {
}
